app/API/V1/Controllers/UserController.php

```diff
@@ -34,6 +38,48 @@ public function getTTConfig(): array
         ];
     }
 
+    /**
+     * @param SimpleEventContract $event
+     */
+    public function onPostStore (SimpleEventContract $event):void
+    {
+        if ($event->getEventArgs()['frontEndOptions']['email'] === true) {
+            $result = $event->getEventArgs()['result'];
+            $query = $event->getEventArgs()['params'];
+
+            $em = app('em');
+
+            /** @var UserRepository $userRepo */
```

Comment:
Much of this should be moved into a method attached to the user repo and also the Email Verification repo